apps/omg_eth/test/omg_eth/client_test.exs

```diff
@@ -0,0 +1,59 @@
+# Copyright 2019-2020 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.Eth.ClientTest do
+  use ExUnit.Case, async: true
+  alias OMG.Eth.Client
+
+  test "get_ethereum_height/0 returns the block number", %{test: test_name} do
```

Comment:
Hmm what's this `%{test: test_name}` sorcery?

Reply:
this is basically ExUnit state